src/Service.php

```diff
@@ -29,6 +29,7 @@ public function __construct(string $apiKey, string $uriBase)
 
         $this->httpClient = new \GuzzleHttp\Client();
         $this->headers['apikey'] = $this->apiKey;
```

Comment:
its been a while since ive used supabase, but is this line needed? i cant remember any use of an apikey header

Reply:
Yes, without it you'll get

```
{"message":"No API key found in request","hint":"No `apikey` request header or url param was found."}
```